xmodule/library_content_block.py

```diff
@@ -97,23 +104,31 @@ class LibraryContentBlock(
 
     show_in_read_only_mode = True
 
-    # noinspection PyMethodParameters
-    @classproperty
-    def completion_mode(cls):  # pylint: disable=no-self-argument
-        """
-        Allow overriding the completion mode with a feature flag.
-
-        This is a property, so it can be dynamically overridden in tests, as it is not evaluated at runtime.
-        """
-        if settings.FEATURES.get('MARK_LIBRARY_CONTENT_BLOCK_COMPLETE_ON_VIEW', False):
-            return XBlockCompletionMode.COMPLETABLE
+    resources_dir = 'assets/library_content'
 
-        return XBlockCompletionMode.AGGREGATOR
+    preview_view_js = {
+        'js': [],
+        'xmodule_js': resource_string(__name__, 'js/src/xmodule.js'),
+    }
+    preview_view_css = {
+        'scss': [],
+    }
+    mako_template = 'widgets/metadata-edit.html'
+    studio_js_module_name = "VerticalDescriptor"
+    studio_view_js = {
+        'js': [
+            resource_string(__name__, 'js/src/vertical/edit.js'),
+        ],
+        'xmodule_js': resource_string(__name__, 'js/src/xmodule.js'),
+    }
+    studio_view_css = {
+        'scss': [],
+    }
 
     display_name = String(
         display_name=_("Display Name"),
         help=_("The display name for this component."),
-        default="Randomized Content Block",
+        default="Library Reference Block",
```

Comment:
```suggestion
        default="Libraries Component",
```
I believe @jmakowski1123 and Brad (GH username?) talked and decided that the new name of this component should just be "Libraries". Given that, do you guys have thoughts on what the default title of the component should be? (This is the title that shows up when you first create a Libraries component, but haven't given it a title yet. Currently it's "Randomized Content Block", but we could call it anything.)

Reply:
Asked in slack https://openedx.slack.com/archives/C05NT2YN820/p1701714420089529

Reply:
@connorhaugh We landed on "Library Content" for the default title, can you make that change here?